### ISSUE TITLE
AuthMethodPickerActivity layout issue

### ISSUE THREAD

**Issue by dimipaun** [firebase/FirebaseUI-Android#1483]
Testing on a Samsung S9+ (as well as a few other phones), we see an empty picker screen (except for the TOS and PP statement at the bottom). If I tweak the layout a bit:

```
diff --git a/auth/src/main/res/layout/fui_auth_method_picker_layout.xml b/auth/src/main/res/layout/fui_auth_method_picker_layout.xml
index 4bac31ba..8eb77a87 100644
--- a/auth/src/main/res/layout/fui_auth_method_picker_layout.xml
+++ b/auth/src/main/res/layout/fui_auth_method_picker_layout.xml
@@ -33,10 +33,11 @@
 
     <ScrollView
         android:id="@+id/container"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/fui_field_padding_vert"
         android:fillViewport="true"
+        android:background="#bfbfbf"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/logo"
```

Then I get this result:
![screenshot_20181011-213852_youkno](https://user-images.githubusercontent.com/625256/46843041-eed0fa80-cd9e-11e8-9015-a2331aac778a.jpg)

So there seems to be an issue with the layout. I'm not a Android layout expert, any thoughts?

**Comment by robapaul88** [comment]
https://github.com/robapaul88/FirebaseUI-Android/commit/cd293b53a4f7d6ac54ba9fcb1f7ec0ab43979830

This issues only happens while in portrait, and the fix seems to be adding:
android:layout_width="wrap_content"
android:layout_height="0dp"
to the logo ImageView in fui_auth_method_picker_layout.xml (basically redeclaring same layout_widht and layout_height properties from @style/FirebaseUI.AuthMethodPicker.Logo in the xml).

**Comment by samtstern** [comment]
@robapaul88 since your fix is harmless I think I will include it in 4.3.0

**Comment by samtstern** [comment]
Speculative fix in #1548.  If anyone can repro this once `4.3.0` comes out we can discuss more.

**Comment by samtstern** [comment]
This has been released in version `4.3.0`.
